skin/frontend/base/default/css/TIG/MyParcel2014/myparcel_options.css

```diff
@@ -186,6 +209,30 @@
     padding: 5px 0px 5px 0px;
 }
 
+.mypa-bold-price{
+	font-weight: bold;
+}
+@media only screen and (max-width: 600px) {
```

Comment:
Er is ook iets in js betreft width van een scherm. Dat moet dan ook verplaatsen naar css. Lukt dat? Want ik heb het liever niet in js en css.